sw/device/lib/testing/test_rom/test_rom.c

```diff
@@ -122,7 +128,33 @@ bool rom_test_main(void) {
     LOG_INFO("Jitter is enabled");
   }
 
+#if !OT_IS_ENGLISH_BREAKFAST
+  // Check the otp to see if flash scramble should be enabled.
+  CHECK_DIF_OK(dif_otp_ctrl_init(
+      mmio_region_from_addr(TOP_EARLGREY_OTP_CTRL_CORE_BASE_ADDR), &otp_ctrl));
+
+  uint32_t otp_val;
+  otp_val = abs_mmio_read32(
+      TOP_EARLGREY_OTP_CTRL_CORE_BASE_ADDR + OTP_CTRL_SW_CFG_WINDOW_REG_OFFSET +
+      OTP_CTRL_PARAM_CREATOR_SW_CFG_FLASH_DATA_DEFAULT_CFG_OFFSET);
+
+  // very stupid, just set the entire first bank to scrambled right now
```

Comment:
Should this comment be tagged with a TODO? I believe @alphan is going to make some updates in the ROM as well.

Reply:
Yeah, but that change might take some time because @cfrantz pointed out that the next owner should be able to control it and we should consider putting these in the info pages. I think we will also have OTP values for bootstrapping this configuration when we don't have anything in the info pages.

@tjaychen I didn't read the rest of the PR but I'm curious: why just the first bank? Can you set the flash_data_default_cfg in the OTP images to scramble everything and just apply that setting here?

Reply:
wait...isn't that what i did? 
o crap sorry, originally i did something else and applied to only the 1st bank.  But I forgot to update the comment afterwards, i'll fix that.

Reply:
updated comment